src/react-components/ui-root.js

```diff
@@ -1671,7 +1672,12 @@ class UIRoot extends Component {
               history={this.props.history}
               render={() =>
                 this.renderDialog(RoomSettingsDialog, {
-                  initialSettings: { name: this.props.hubName, member_permissions: this.props.hubMemberPermissions },
+                  showPrivacySettings: configs.isAdmin() || configs.feature("public_rooms"),
```

Comment:
we should revisit this -- this probably needs to be a permission check and not config driven. the reticulum server should carve out and return a permission for being able to make rooms public.